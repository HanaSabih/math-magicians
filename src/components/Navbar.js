// import { Outlet, Link } from 'react-router-dom';
import { Outlet } from 'react-router-dom';
import Container from 'react-bootstrap/Container';
import Nav from 'react-bootstrap/Nav';
import Navbar from 'react-bootstrap/Navbar';

const Navbars = () => (
  <>
    <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
      <Container>
        <Navbar.Brand href="/">Math magicians</Navbar.Brand>
        <Navbar.Toggle aria-controls="responsive-navbar-nav" />
        <Navbar.Collapse id="responsive-navbar-nav">
          <Nav className="me-auto justify-content-end flex-grow-1 pe-3">
            <Nav.Link href="/">Home</Nav.Link>
            <Nav.Link href="/Calculator">Calculator</Nav.Link>
            <Nav.Link href="/Quote">Quote</Nav.Link>
          </Nav>

        </Navbar.Collapse>
      </Container>
    </Navbar>
    <Outlet />
  </>
 
);

export default Navbars;
