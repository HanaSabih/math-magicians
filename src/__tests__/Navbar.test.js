import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

it('renders correctly', () => {
  const testQuote = renderer.create(<Navbar />).toJSON();
  expect(testQuote).toMatchSnapshot();
});