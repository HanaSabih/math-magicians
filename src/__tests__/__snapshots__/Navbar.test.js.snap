// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Math magicians
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        className="me-auto justify-content-end flex-grow-1 pe-3 navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rr-ui-event-key="/"
          disabled={false}
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="/Calculator"
          disabled={false}
          href="/Calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="/Quote"
          disabled={false}
          href="/Quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </div>
    </div>
  </div>
</nav>
`;
